behave/features/cla_fala/fala_end_to_end_tests.feature

```diff
@@ -25,9 +25,10 @@ Feature: FALA end to end tests
     When I browse through the filter categories and select "<filter_label>"
     And I select the 'Apply filter' button
     Then the result page containing "<location>" is updated to apply the filter "<filter_label>"
+    And 10 result is visible on the results page
```

Comment:
as discussed - why is this looking for 10? 
answer this is one page worth of results
this step assumes that there will always be more than 10 available so at least one full page of results
reuses another step hence specifying 10
would like some sort of comment in the feature file to explain why looking for 10 in this particular use case rather than just in the PR

Reply:
Maybe rename step to explain that it's the top 10 results for the google maps plugin. Or as Joe says add a comment above the feature file line explaining the 10 results.

Reply:
I don't think we need this step. Can we not use `There are less results visible on the results page` instead here? Capture the search result number before applying the filter on the search result page and then compare?

Reply:
Just thinking we're doing two different results compare, one with the google maps and one with the results number in the paragraph. I think I prefer the way we're doing it in your new step.